src/transliteration/ui/SubIndex.tsx

```diff
@@ -14,7 +14,9 @@ export const subscriptNumbers: ReadonlyMap<string, string> = new Map([
   ['9', '₉'],
 ])
 
-function numberToUnicodeSubscript(number: number | null | undefined): string {
+export function numberToUnicodeSubscript(
```

Comment:
Move this to the application or domain. Do not export utility functions from components.